src/views/ajax.js

```diff
@@ -0,0 +1,112 @@
+const util = require('../lib/util');
+const dom = require('../lib/dom');
+
+let count = 0;
+let ready = false;
+let preRenderCache = [];
+let currentTimout = undefined;
+const summaryStack = [];
+
+function rowTemplate(method, uri, duration) {
+    return `
+        <div class="glimpse-ajax-row">
+            <span class="glimpse-section-label">${method}</span>
+            <span class="glimpse-ajax-uri" title="${uri}">${uri}</span>
+            <span>
+                <span>${duration}</span>
+                <span class="glimpse-section-label">ms</span>
+            </span>
+        </div>
+    `;
+}
+
+function update(method, uri, duration, size, status, statusText, time, contentType, requestId) {
+    count++;
+
+    //manage counter value
+    var counter = document.getElementById('glimpse-ajax-count');
+    counter.innerText = count;
+    dom.addClass(counter, 'glimpse-section-value--update');
+    if (currentTimout) {
+        clearTimeout(currentTimout);
+    }
+    currentTimout = setTimeout(function() {
+        dom.removeClass(counter, 'glimpse-section-value--update');
+    }, 2000);
+
+    //manage row values
+    if (summaryStack.length === 0) {
+        var section = document.getElementById('glimpse-ajax');
+        section.insertAdjacentHTML('beforeend', '<div class="glimpse-ajax-rows" id="glimpse-ajax-rows"></div>');
```

Comment:
Thought about that and was trying to optimize for the case where there are no ajax requests. I could go either way though.